datasets/kinnews_kirnews/README.md

```diff
@@ -0,0 +1,167 @@
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** [Needs More Information]
+- **Repository:** https://github.com/Andrews2017/KINNEWS-and-KIRNEWS-Corpus
+- **Paper:** [KINNEWS and KIRNEWS: Benchmarking Cross-Lingual Text Classification for Kinyarwanda and Kirundi](https://arxiv.org/abs/2010.12174)
+- **Leaderboard:** NA
+- **Point of Contact:** [Rubungo Andre Niyongabo1](mailto:niyongabor.andre@std.uestc.edu.cn)
+
+### Dataset Summary
+
+Kinyarwanda and Kirundi news classification datasets (KINNEWS and KIRNEWS,respectively), which were both collected from Rwanda and Burundi news websites and newspapers, for low-resource monolingual and cross-lingual multiclass classification tasks.
+
+### Supported Tasks and Leaderboards
+
+NA
+
+### Languages
+
+Kinyarwanda and Kirundi
+
+## Dataset Structure
+
+### Data Instances
+
+Each dataset is in comma-separated-value (csv) format, with columns that are described bellow (Note that in the cleaned versions we only remain with 'label','title', and 'content' columns):
+| Field | Description |
+| ----- | ----------- |
+| label | Numerical labels that range from 1 to 14 |
+| en_label | English labels |
+| kin_label | Kinyarwanda labels |
+| kir_label | Kirundi labels |
+| url | The link to the news source |
+| title | The title of the news article |
+| content | The full content of the news article |
+
+### Data Fields
+
+The raw version of the data for Kinyarwanda language consists of these fields
+-label: The category of the news article
+-kin_label: The associated label in Kinyarwanda language
+-en_label: The associated label in English
+-url: The URL of the news article
+-title: The title of the news article
+-content: The content of the news article
+
+The cleaned version contains only the `label`, `title` and the `content` fields
+ 
+
+### Data Splits
+
+Lang| Train | Test |
+|---| ----- | ---- |
+|Kinyarwanda|17014|4254|
+|Kirundi|3689|923|
+
+## Dataset Creation
+
+### Curation Rationale
+
+[Needs More Information]
```

Comment:
```suggestion
[More Information Needed]
```

Can you use `[More Information Needed]` instead ?
That's what we're using in all other dataset cards

Reply:
Changed the string to "More Information Needed" consistently